Sources/Classes/core/Models/MiniAppJavaScriptMessageInfo.swift

```diff
@@ -52,6 +53,10 @@ struct MiniAppCustomPermissionsListResponse: Codable {
     let status: String
 }
 
+struct MiniAppShouldClose: Decodable {
+    let withConfirmation: Bool?
+}
+
```

Comment:
I'm not sure about the name of this struct, maybe something like `MiniAppCloseRequest`

Reply:
This is not required hence removed it in latest commits.